--- tests/components/FulfillmentOrderSearch.spec.ts
+++ tests/components/FulfillmentOrderSearch.spec.ts
@@ -31,6 +31,8 @@
     data: {
       dataDocumentId: 'SHOPIFY_SHOP_ORDER_SEARCH',
       customParametersMap: { shopId: 'shop', orderName_op: 'contains', orderName_ic: 'Y', orderName: '#10', orderByField: '-orderDate' },
+      pageIndex: 0,
+      pageSize: 21
     },
   });
   const more = w.findAllComponents(IonButton).find(row => row.text() === 'More results')!;
